libs/src/api/account.js

```diff
@@ -487,6 +488,22 @@ class Account extends Base {
     this.REQUIRES(Services.IDENTITY_SERVICE)
     return this.ethContracts.AudiusTokenClient.transferFrom(owner, address, relayer, amount)
   }
+
+  /**
+   * Updates the minimum delegation amount for a user in identity
+   * NOTE: Requests eth account signature
+   */
+  async updateMinimumDelegationAmount (amount) {
+    this.REQUIRES(Services.IDENTITY_SERVICE)
+    const unixTs = Math.round(new Date().getTime() / 1000) // current unix timestamp (sec)
+    const message = `Click sign to authenticate with identity service: ${unixTs}`
```

Comment:
Was not sure if I should put in this libs or take it out and just pass in the message & signature - I think it depends if we want to customize the message

Reply:
seems fine